components/docs-chef-io/content/automate/ha_chef_backend_to_automate_ha.md

```diff
@@ -17,276 +15,334 @@ gh_repo = "automate"
-```
+    ```cmd
+        hab pkg exec chef/knife-ec-backup knife ec backup backup_$(date '+%Y%m%d%H%M%s') --webui-key /etc/opscode/webui_priv.pem -s <chef server url>
+    ```
 
--  Execute the below command to clean unused data from reports. This is an optional steps
+    For example:
 
-``` bash
-    hab pkg exec chef/knife-ec-backup knife tidy server clean --backup-path /path/to/an-ec-backup
-```
+    ```cmd
+        hab pkg exec chef/knife-ec-backup knife ec backup backup_$(date '+%Y%m%d%H%M%s') --webui-key /etc/opscode/webui_priv.pem -s https://chef.io`.
+    ```
 
-5.   Execute the below command to copy the backup directory to the Automate HA Chef Server.
-```cmd
-     scp -i /path/to/key backup\_$(date '+%Y%m%d%H%M%s') user@host:/home/user
-```
+    - Execute the below command to clean unused data from reports. This is an optional step
+
+    ```bash
+        hab pkg exec chef/knife-ec-backup knife tidy server clean --backup-path /path/to/an-ec-backup
+    ```
+
+5. Execute the below command to copy the backup directory to the Automate HA Chef Server.
+
+    ```cmd
+        scp -i /path/to/key backup\_$(date '+%Y%m%d%H%M%s') user@host:/home/user
+    ```
+
+    If your HA Chef Server is in a private subnet, scp backup file to bastion and then to Chef Server.
 
 ## Restore Backed Up Data to Chef Automate HA
 
--   Execute the below command to install the habitat package for `knife-ec-backup`
+- Execute the below command to install the habitat package for `knife-ec-backup`
 
-```cmd
-    hab pkg install chef/knife-ec-backup
-```
+    ```cmd
+        hab pkg install chef/knife-ec-backup
+    ```
 
--   Execute the below command to restore the backup.
+- Execute the below command to restore the backup.
 
-```cmd
-    hab pkg exec chef/knife-ec-backup knife ec restore /home/centos/backup\_2021061013191623331154 -yes --concurrency 1 --webui-key /hab/svc/automate-cs-oc-erchef/data/webui\_priv.pem --purge -c /hab/pkgs/chef/chef-server-ctl/*/*/omnibus-ctl/spec/fixtures/pivotal.rb
-```
+    ```cmd
+        hab pkg exec chef/knife-ec-backup knife ec restore <path/to/directory/backup_file> -yes --concurrency 1 --webui-key /hab/svc/automate-cs-oc-erchef/data/webui\_priv.pem --purge -c /hab/pkgs/chef/chef-server-ctl/*/*/omnibus-ctl/spec/fixtures/pivotal.rb
+    ```
 
-## In place Migration (Chef Backend to Automate HA)
+## Steps to validate if Migration is successful
 
-As part of this scenario, customer will migrate from chef-backend (5 machines) to Automate HA in-place, i.e. Automate HA will be deployed in those 5 machines only where Chef-backend is running. One extra bastion node will be required which will be managing the deployment of Automate HA on the chef backend infrastructure.
+- Execute the below command from the old server where a knife is installed and from the new server where the knife is installed :
```

Comment:
Let's be more explicit and split this statements into two steps:
- Fetching for older server. 
  <commad>
- Fetching from HA
<command>
-  Check the difference